src/TestProblem/ELBDM/LSS/Init_TestProb_ELBDM_LSS.cpp

```diff
@@ -188,7 +189,20 @@ void Init_ByFile_ELBDM_LSS( real fluid_out[], const real fluid_in[], const int n
                     "LSS_InitMode", LSS_InitMode );
    } // switch ( LSS_InitMode )
 
-   fluid_out[DENS] = SQR( fluid_out[REAL] ) + SQR( fluid_out[IMAG] );
+
+#  if ( ELBDM_SCHEME == ELBDM_HYBRID )
+   if ( amr->use_wave_flag[lv] ) {
+#  endif
+   fluid_out[DENS] = SQR( Re ) + SQR( Im );
+   fluid_out[REAL] = Re;
+   fluid_out[IMAG] = Im;
+#  if ( ELBDM_SCHEME == ELBDM_HYBRID )
+   } else { // if ( amr->use_wave_flag[lv] )
+   fluid_out[DENS] = Re;
+   fluid_out[PHAS] = Im;
+   fluid_out[STUB] = 0.0;
+   } // if ( amr->use_wave_flag[lv] ) ... else
+#  endif
```

Comment:
@hyschive 
You are right. I fixed the bug and updated the code in order for its intention to be clearer.

Reply:
@hyschive 
Upon closer reflection, I think it would be best to really just load Dens/Phas on ALL levels for the hybrid scheme and not switch. We can always unambiguously convert back to Re/Im and I imagine it will be easier to design initial conditions that way. What do you think?

Reply:
@KunkelAlexander Sorry for the late response. Somehow I missed this message. I like your current approach, which is much clearer!